tests/household.test.js

```diff
@@ -0,0 +1,36 @@
+import request from 'supertest';
+
+import { ALL_PAGEABLE_ENDPOINTS, ALL_GEN_ENDPOINTS } from "./endpoints.js";
+
+// Basic helpers
+function cosi_request() {
+    return request("127.0.0.1:8000");
+}
+
+function expectKeys(json_data, keys) {
+    expect(Object.keys(json_data)).toEqual(keys);
+}
+
+// Test setup
+beforeAll(async ()=> {
```

Comment:
I think it makes sense to move this into a separate file instead of in household. Something like conftest.js? Same with Cosi connection, but that can probably go into a common.

Reply:
I agree. If it's OK with you, I think I'd like to tackle this in the next PR.